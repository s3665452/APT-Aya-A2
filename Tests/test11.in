YBUUYRURRBRLRULLLLLBBRYRLBYURUBULYRULYBYBYRLYLUYUYBYBBUURLLBURRLLBLBLULBYRRBUUYYRYYBBRYUUYUBYBLRLRRU
Eve
El
turn 1 U 2
turn 5 L 3
turn 4 R 1
turn 3 R 4
turn 0 B 3
turn 2 R 4
turn 0 L 4
turn 0 Y 2
turn 0 U 5
turn 3 U 5
turn 1 R 2
turn 2 L 4
turn 0 B 3
turn 4 U 5
turn 0 Y 4
turn 5 Y 2
turn 0 R 1
turn 0 U 1
turn 0 L 5
turn 0 B 6
turn 5 L 5
turn 4 B 2
turn 3 Y 4
turn 0 U 3
turn 2 L 5
turn 1 B 1
turn 0 Y 3
turn 0 B 4
turn 0 L 1
turn 0 R 5
turn 0 U 2
turn 1 R 5
turn 3 U 2
turn 2 B 4
turn 0 L 4
turn 4 R 5
turn 0 Y 1
turn 5 Y 3
turn 0 U 3
turn 0 B 6
turn 1 Y 5
turn 3 Y 3
turn 5 R 3
turn 2 U 4
turn 4 R 3
turn 0 U 4
turn 0 B 4
turn 0 R 2
turn 0 L 2

