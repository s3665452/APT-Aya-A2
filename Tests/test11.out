Factories:
0: Y Y
1:
2:
3:
4:
5:

Score for Player Eve: 18
Mosaic for Eve:
1:         . || B . R U .
2:       R R || . B Y . U
3:     Y Y Y || U . B . .
4:   U U U U || . . L B .
5: . . . . . || . R U . .
broken: F

Score for Player El: 32
Mosaic for El:
1:         . || . Y R . L
2:       L L || . . Y R U
3:     R R R || U L B Y .
4:   B B B B || R . L . Y
5: . . . Y Y || . . . L .
broken:

